lampshade.go

```diff
@@ -187,41 +222,45 @@ const (
 	winSize        = 4
 	maxPaddingSize = 1
 	tsSize         = 8
+	maxSecretSize  = 32
```

Comment:
The actual secretSize depends on the specific cipher, which is why I named this maxSecretSize.